### ISSUE TITLE
Backend needs documentation on how to run...

### ISSUE THREAD

**Issue by ytimocin** [tuyendev/mee-spring-vue-template#2]
This repo looks interesting. Would you mind adding some more documentation on how to run the backend too? I would also like to contribute if possible.

**Comment by jonaskahn** [comment]
@ytimocin Yeah, sure thing. Sorry, I didn't have time to document yet but I will write the document detail soon when I finish my work.
